File: ESAMI/DOCS_NOTES/Fondamenti di Informatica II/AlgoritmiEStruttureDati-master/AlgoritmiEStruttureDati-master/Algoritmi - Esami/ALGORITMI/giugno2017/java-aux/Driver.java
public class Driver {


    public static void main(String[] argv) {

    	BST b = new BST();
    	
        b.BST_insert(3);
        b.BST_insert(12);
        b.BST_insert(7);
        b.BST_insert(5);
        b.BST_insert(1);
        b.BST_insert(7);
        b.BST_insert(15);
        b.BST_insert(1);
        b.BST_insert(8);
        b.BST_insert(7);
        b.BST_insert(12);

        System.out.println("Albero: \n");
        b.BST_print();

        System.out.println("\nRicerca valore: \n");
        for (int k=0; k<20; k++){
            if (b.BST_find(k)==null)
                System.out.println("Nodo con chiave "+k+" NON trovato"); 
            else
                System.out.println("Nodo con chiave "+k+" trovato"); 
        }

        System.out.println("\nRicerca massimo: \n");
        for (int level=1; level<20; level++){
            int max = b.maxOnLevel(level);

            if (max == -1)
            	System.out.println("Livello "+level+" vuoto");
            else
            	System.out.println("Valore massino nel livello "+level+": "+ max);
        }
    }
}

File: ESAMI/DOCS_NOTES/Fondamenti di Informatica II/AlgoritmiEStruttureDati-master/AlgoritmiEStruttureDati-master/Algoritmi - Esami/ALGORITMI/luglio2016/c-aux/driver.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include "bst.h"
#include "queue.h"


int main(int argc, char** argv) {

        BST * b = BST_new();

        BST_insert(b, 3);
        BST_insert(b, 12);
        BST_insert(b, 7);
        BST_insert(b, 5);
        BST_insert(b, 1);
        BST_insert(b, 7);
        BST_insert(b, 15);
        BST_insert(b, 1);
        BST_insert(b, 8);
        BST_insert(b, 7);
        BST_insert(b, 12);

        printf("Albero: \n");
        BST_print(b);

        BST * b2 = BST_new();

        BST_insert(b2, 13);
        BST_insert(b2, 5);
        BST_insert(b2, 20);
        BST_insert(b2, 3);
        BST_insert(b2, 7);
        BST_insert(b2, 25);
        BST_print(b2);

   /*     printf("\nRicerca valore: \n");
        for (int k=0; k<20; k++){
            if (BST_find(b,k)==NULL)
                printf("Nodo con chiave %d NON trovato\n", k);
            else
                printf("Nodo con chiave %d trovato\n", k);
        }*/


        printf("\nSottoinsieme tra due BST\n");
            int x = sottoinsieme(b, b2);
            if(x == 0) printf("b1 non sottoinisime di b2 e viceversa\n");
            else printf("b1 o b2 sottoinsieme dell'altro\n");
 /*       printf("\nRicerca massimo: \n");
        for (int level=1; level<20; level++){
            int max = maxOnLevel(b,level);

            if (max == -1)
                printf("Livello %d vuoto\n", level);
            else
                printf("Valore massino nel livello %d: %d\n",level, max);
        }*/

        BST_delete(b);
}
